roles/keycloak/tasks/main.yml

```diff
@@ -240,7 +240,7 @@
 - name: Ensure Keycloak service is active and enabled on boot
   systemd:
     name: "keycloak"
-    state: "started"
+    state: "restarted"
```

Comment:
Can we leave this as "started"? We should use notify Restart/reload keycloak for any task that requires restarting the service

Reply:
This is there in case we already have a running instance and we just replaced every file of the service in the filesystem, and we need to ensure it's restarted before we perform the configuration

Reply:
Whenever you make any change in the filesystem - in case of upgrade, we actually replace everything, so, a restart is mandatory.